lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -187,6 +187,14 @@
     - ' - C(runonce) (list): List of commands to run at first user logon.'
     - ' - C(timezone) (int): Timezone (See U(https://msdn.microsoft.com/en-us/library/ms912391.aspx)).'
     version_added: '2.3'
+  action:
```

Comment:
we try to avoid `action` as it is a bad pattern for the declarative voice we want to keep in modules, it is clearly imperative.

i would implement clone via a `present` state and a clone source, possibly with a force option to clobber existing guests

as for powerstate .. isn't this handled already with states `poweredon` and `poweredoff`?